communication/README.md

```diff
@@ -146,22 +146,22 @@ Office hours are held once a month. Please refer to [the docs][office-hours]
 to learn more. Join the conversation on slack under [#office-hours].
 
 ### Other
-- [r/kubernetes] - reddit channel owned and operated by community members and
-not an official channel for the project.
+- [r/kubernetes] - reddit channel is owned and operated by community members and
+is not an official channel for the project.
```

Comment:
again, this isn't an improvement.  the original was a definition, that is a sentence fragment intended to describe the thing.  adding a verb here is just confusing, since it's still an incomplete sentence, but now it looks like it ought to be a real sentence.